tests/src/main.rs

```diff
@@ -87,42 +88,46 @@ fn main() -> io::Result<()> {
         expected.display(),
         &inner
       );
-    }
-
-    let diff_output_result = Command::new("colordiff")
-      .arg("--unified")
-      .args(["--label", "output"])
-      .arg(output)
-      .args(["--label", "expected"])
-      .arg(expected)
-      .output();
-
-    let diff_output = match diff_output_result {
-      Ok(o) => o,
-      Err(e) => {
-        eprintln!("Could not run colordiff: attempt failed with \"{e}\"\nIs colordiff installed and available executable on $PATH ?");
-        return Err(e);
-      }
-    };
-
-    if diff_output.status.success() {
-      eprintln!("ok");
-      passed += 1;
     } else {
-      if diff_output.status.code() == Some(2) {
-        eprintln!("syntax highlightning mismatch:");
+      let diff_output_result = Command::new("colordiff")
```

Comment:
Instead of an else branch, the error print above should call continue to skip the current file. That will leave the rest of the code unindented.